### DISCUSSION TITLE
6/5/24

### DISCUSSION THREAD

**Discussion by Pamela198713** [enbalde-ispc/enbalde-ispc#106] [Daily Sprint]
Que hice ayer?
 Que hago hoy?
Impedimento?

**Comment by Pamela198713** [comment]
Buenas! Feliz inicio de semana...
Ayer no hice nada recibí visitas de FLIA de España 
Hoy nos reunimos a las 20 hs para organizar actividades de otros módulos. 
Impedimento, el tiempo y el cansancio del cursado..

**Comment by rpgrca** [comment]
Ayer creo que terminé con la navegación de la app, hoy voy a estar más ocupado en el trabajo y a la noche voy a juntarme con el grupo a las 20/22 para las demás actividades. Me gustaría ver si puedo empezar a implementar el carrito de compras, que aparezcan listados lo que se selecciona, que se puedan subir y bajar items y que calcule el total antes del miércoles.

**Comment by LeonardoEC** [comment]
Ayer me puse con leer lo teórico de emprendedurismo 
Hoy se trabaja todo el día 
Estoy agobiado y estresado

**Comment by LuciaL152** [comment]
Ayer no hice nada porque no me sentía muy bien.
Hoy estuve viendo los materiales de estudio.
El tiempo.
